export const getErrorCause = () => new Error('This is a error cause');
